pkg/beeclient/api/chunks.go

```diff
@@ -33,6 +32,6 @@ func (c *ChunksService) Upload(ctx context.Context, a swarm.Address, data io.Rea
 	if o.Pin {
 		h.Add("Swarm-Pin", "true")
 	}
-	err := c.client.requestWithHeader(ctx, http.MethodPost, "/"+apiVersion+"/chunks/"+a.String(), h, data, &resp)
+	err := c.client.requestWithHeader(ctx, http.MethodPost, "/"+apiVersion+"/chunks", h, data, &resp)
```

Comment:
It is no longer needed to pass the address to ChunksService.Upload (this) method. I would even say that data as io.Readed could be changed to []byte type as the chunk size is very small, but this is just a minor detail, if you want to consider.